frontend/src/app/[locale]/auth/_components/SignUpForm.tsx

```diff
@@ -1,7 +1,112 @@
-import React from "react";
+"use client";
+
+import { zodResolver } from "@hookform/resolvers/zod";
+import { useForm } from "react-hook-form";
+import { z } from "zod";
+
+import { Button } from "@/components/ui/button";
+import {
+	Form,
+	FormControl,
+	FormField,
+	FormItem,
+	FormLabel,
+	FormMessage,
+} from "@/components/ui/form";
+import { Input } from "@/components/ui/input";
+import { useTranslations } from "next-intl";
 
 const SignUpForm = () => {
-	return <div>SignUpForm</div>;
+	const t = useTranslations("SignUpForm");
+	// TODO: Add password regex
+	const formSchema = z
+		.object({
+			email: z
+				.string()
+				.min(5, {
+					message: t("emailTooShortError"),
+				})
```

Comment:
Check redundant. Any valid email cannot be shorter that 5 chars

Reply:
temporary, until regex is done

Reply:
feat(frontend): [PYTA-16] add signup page *template*

Reply:
yes, but why add something that should never be there, also [https://emailregex.com/](https://emailregex.com/)